mods/persistence/datums/extensions/extensions.dm

```diff
@@ -17,4 +17,11 @@
 /datum/extension/should_save(object_parent)
 	if(object_parent) // Extensions are saved manually, either by self-reporting or by the one off serializer checking. Don't permit saving from object vars.
 		return FALSE
-	return should_save
\ No newline at end of file
+
+	// If the holder is a movable and wouldn't be saved, don't save this either.
+	if(istype(holder, /atom/movable))
```

Comment:
Side note, don't merge this yet, I forgot to add a check for limbo. Won't get to it for a couple days most likely.

Reply:
Should work now.